CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+## Testing
+
+### Running tests
+
+Use the `rake` command to run the entire test suite. This runs on docker due to a dependency on Envoy.
+
+### Unit tests
+
+Unit tests can be run without docker by running `rspec spec/unit`
```

Comment:
I think it'd be nice to come back and make the `contributing.md` split into parts. Local setup/testing and contributing guidelines to the repo. Ex: https://github.com/puma/puma/blob/master/CONTRIBUTING.md